pkg/tecdsa/signing/gen/pb/message.proto

```diff
@@ -20,4 +20,10 @@ message TSSRoundTwoMessage {
     uint32 senderID = 1;
     map<uint32, bytes> peersPayload = 2;
     string sessionID = 3;
+}
+
+message TSSRoundThreeMessage {
+    uint32 senderID = 1;
+    bytes payload = 2;
+    string sessionID = 3;
 }
```

Comment:
It seems the file is not formatted with `go fmt` (e.g. there is no newline a the end of the file).
I would consider formatting them.